src/zadanie2/TextCheck.java

```diff
@@ -0,0 +1,33 @@
+package zadanie2;
+
+public class TextCheck {
+    private static long countCharsInText(String text) {
+        return text.chars().count();
+    }
+
+    private static long countCharsInTextWithoutSpaces(String text) {
+        return text.replace(" ", "").chars().count();
+    }
+
+    private static long countWordsInText(String text) {
+        return text.trim().split(" ").length;
+    }
+
+    private static boolean isTextPalindrome(String text) {
+        text = text.toLowerCase();
```

Comment:
Staraj się nie modyfikować parametrów które dostajesz w metodzie. Dobrym przyzwyczajeniem jest dawać przed parametrem słowo `final`. Taki kod jest łatwiejszy do zrozumienia jak czytasz. W tym przypadku jest to pomijalne, ale przy większych metodach taki zabieg jest przydatny.